my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
```

Comment:
For PEP-8 compliance there should be a line-break here, and more generally throughout the code for readability.